cmake.deps/cmake/BuildTreesitterParsers.cmake

```diff
@@ -23,3 +23,4 @@ endfunction()
 BuildTSParser(c ${TREESITTER_C_URL} ${TREESITTER_C_SHA256} TreesitterParserCMakeLists.txt)
 BuildTSParser(lua ${TREESITTER_LUA_URL} ${TREESITTER_LUA_SHA256} TreesitterParserCMakeLists.txt)
 BuildTSParser(vim ${TREESITTER_VIM_URL} ${TREESITTER_VIM_SHA256} TreesitterParserCMakeLists.txt)
+BuildTSParser(help ${TREESITTER_HELP_URL} ${TREESITTER_HELP_SHA256} TreesitterParserCMakeLists.txt)
```

Comment:
Just for the record (in case someone needs it later): the first `help` here is what's responsible for the name of the `.so` file; the CMake variable name can be whatever you want (so leaving those as `VIMDOC` would have been fine).